checkST2110.js

```diff
@@ -28,12 +28,14 @@ const groupPattern = /a=group:DUP\s+(\S+)\s+(\S+)/;
 const ssrcPattern = /a=ssrc:(\d+)\s/;
 const videoPattern = /video\s+(\d+)(\/\d+)?\s+(RTP\/S?AVP)\s+(\d+)/;
 const rtpmapPattern = /a=rtpmap:(\d+)\s(\S+)\/(\d+)\s*/;
+const bandwidthPattern = /b=([a-zA-Z]+):(\d*\s*$)/;
```

Comment:
I corrected the digit check for bandwidth and removed possibility of any whitespace after as per 4566. I left the check for alphabetic chars since it's a 2110 check and only allowed values are from that set.  If we need to tighten up the test to allow checks for all token chars permitted by 4566 then probably should be added down in the set of tests for RFC4566.